ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -277,12 +278,12 @@ initBlockFetchConsensusInterface cfg chainDB getCandidates blockFetchSize
     plausibleCandidateChain :: AnchoredFragment (Header blk)
                             -> AnchoredFragment (Header blk)
                             -> Bool
-    plausibleCandidateChain = preferCandidate cfg
+    plausibleCandidateChain = preferAnchoredCandidate cfg
 
     compareCandidateChains :: AnchoredFragment (Header blk)
                            -> AnchoredFragment (Header blk)
                            -> Ordering
-    compareCandidateChains = compareCandidates cfg
+    compareCandidateChains = compareAnchoredCandidates cfg
```

Comment:
(Not necessarily in this PR:) could the BlockFetch client call our `preferCandidate` and `compareCandidates` directly? Maybe that might simplify the BFC implementation? See https://github.com/input-output-hk/ouroboros-network/blob/a7b3055c1a4ccc26455939910a6283d8947c108e/ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision.hs#L136

Reply:
https://github.com/input-output-hk/ouroboros-network/issues/1382